app/pages/lab/visibility.cjsx

```diff
@@ -211,4 +211,5 @@ module.exports = React.createClass
           </div>
       }</PromiseRenderer>
       <p className="form-help">In a live project active workflows are available to volunteers and cannot be edited. Inactive workflows can be edited if a project is live or in development.</p>
+      <p className="form-help">On a live project, if you want to switch which subjects sets are associated with an active workflow: set the workflow to inactive, next change which subject sets are linked in the Workflow Section within the Project Builder, then return the workflow to active.</p>
```

Comment:
@camallen if that's doable that would be easier and more straight forward I think on the user side of the project builder. I suggested the text since currently this is the recommended way of switching subject sets, and its not intuitive when you get to the visibility page. See #824, but I think what you suggest is more elegant and less clicks